src/LinkDotNet.Blog.Infrastructure/Persistence/Sql/Mapping/BlogPostConfiguration.cs

```diff
@@ -26,3 +30,14 @@ public void Configure(EntityTypeBuilder<BlogPost> builder)
             .IsDescending(false, true);
     }
 }
+
+
+internal sealed class BlogPostIdGenerator : ValueGenerator<string>
```

Comment:
What is the advantage here over the current approach?

Reply:
I thought short URLs were more search-engine-friendly after a few investigations I realized it does not have any impact on ranking so I guess we can remove this.